backend/auth/src/main.ts

```diff
@@ -5,6 +5,6 @@ import { AppModule } from './app.module';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.useGlobalPipes(new ValidationPipe({ whitelist: true }));
-  await app.listen(3005);
+  await app.listen(3001);
```

Comment:
Ports in env? mit fallback?

Reply:
on it

Reply:
All services now default to process.env.port and fallback to their respective port